index.d.ts

```diff
@@ -1,10 +1,15 @@
 declare function deepmerge<T>(x: Partial<T>, y: Partial<T>, options?: deepmerge.Options): T;
 declare function deepmerge<T1, T2>(x: Partial<T1>, y: Partial<T2>, options?: deepmerge.Options): T1 & T2;
 
+declare interface CustomMerge<T1, T2> {
```

Comment:
This is unused, isn't it?

Reply:
Yes indeed! I'll send up a commit with this taken out

Reply:
Added a commit with this removed